opendatahub-io/odh-dashboard
# Include NIM enablement manifests to the Kustomization files

## Description


Include NIM enablement manifests to the Kustomization files

## How Has This Been Tested?



1. Install Openshift AI
2. Check if NIM is shown in the App list

## Test Impact


No impact on existing code

## Request review criteria:



Self checklist (all need to be checked):

If you have UI changes: 


After the PR is posted & before it merges:
